src/js/containers/LimitOrder/LimitOrder.js

```diff
@@ -128,7 +128,15 @@ export default class LimitOrder extends React.Component {
 
     return {sourceTokenSymbol, sourceToken, destTokenSymbol, destToken}
   }
-
+  async fetchFavoritePairsIfLoggedIn(){
+    if (isUserLogin) {
+      limitOrderServices.getFavoritePairs().then(
```

Comment:
If its async func, please use await with this, code will be clearer

Reply:
Updated, using await